ui/app/components/oidc/assignment-form.js

```diff
@@ -26,6 +26,7 @@ export default class OidcAssignmentFormComponent extends Component {
   @service store;
   @service flashMessages;
   @tracked modelValidations;
+  @tracked errorBanner;
```

Comment:
The assignment form already has a validation against creating an assignment without at least one group or entity so we're it is intentional that we are not setting an inline alert based on a validations error like we are for the other models